### PR TITLE
docs: add 5.15.4 chaneglog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 5.15.4

`2024-03-24`

- 🐞 修复 GetRef 工具类型不能正确获得某些组件 ref 类型的问题。[#47983](https://github.com/ant-design/ant-design/pull/47983)
- 💄 修复 Radio 禁用状态 hover 样式。[#47972](https://github.com/ant-design/ant-design/pull/47972) [@madocto](https://github.com/madocto)
- 🐞 修复 WaterMark 在特定屏幕缩放下会不断重复渲染的问题。[#47895](https://github.com/ant-design/ant-design/pull/47895)
- TypeScript
  - 🤖 导出 AffixRef 类型。[#47982](https://github.com/ant-design/ant-design/pull/47982) [@li-jia-nan](https://github.com/li-jia-nan)

-----------------

## 5.15.4

`2024-03-24`

- 🐞 Fix GetRef ts util can not get correct ref type for some component. [#47983](https://github.com/ant-design/ant-design/pull/47983)
- 💄 Fix Radio disabled hover style. [#47972](https://github.com/ant-design/ant-design/pull/47972) [@madocto](https://github.com/madocto)
- 🐞 Fix WaterMark sometime repeat re-render when browser set scale. [#47895](https://github.com/ant-design/ant-design/pull/47895)
- TypeScript
  - 🤖 export AffixRef interface. [#47982](https://github.com/ant-design/ant-design/pull/47982) [@li-jia-nan](https://github.com/li-jia-nan)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ tag: vVERSION
 
 ---
 
+## 5.15.4
+
+`2024-03-24`
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:21]:
```suggestion
`2024-03-25`
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ tag: vVERSION
 
 ---
 
+## 5.15.4
+
+`2024-03-24`
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:21]:
```suggestion
`2024-03-25`
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -16,6 +16,16 @@ tag: vVERSION
 
 ---
 
+## 5.15.4
+
+`2024-03-25`
+
+- 🐞 修复 GetRef 工具类型不能正确获得某些组件 ref 类型的问题。[#47983](https://github.com/ant-design/ant-design/pull/47983)
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:23]: 这个是不是应该放在下面的 TypeScript 里面

**Comment by li-jia-nan** [comment]
- #48053

File: CHANGELOG.zh-CN.md (hunk 3 above)
Comment (chenshuai2144) [reply CHANGELOG.zh-CN.md:23]: 这个算  bug 吧，改了代码了

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -16,6 +16,17 @@ tag: vVERSION
...
+## 5.15.4
+
+`2024-03-25`
+
+- 💄 Fix QRCode that custom style would be overrided by internal style. [#48053](https://github.com/ant-design/ant-design/pull/48053) [@li-jia-nan](https://github.com/li-jia-nan)
+- 💄 Fix Radio disabled hover style. [#47972](https://github.com/ant-design/ant-design/pull/47972) [@madocto](https://github.com/madocto)
+- 🐞 Fix Watermark sometime repeat re-render when browser set scale. [#47895](https://github.com/ant-design/ant-design/pull/47895)
+- TypeScript
+  - 🤖 Export AffixRef interface. [#47982](https://github.com/ant-design/ant-design/pull/47982) [@li-jia-nan](https://github.com/li-jia-nan)
+  - 🤖 Fix GetRef ts util can not get correct ref type for some component. [#47983](https://github.com/ant-design/ant-design/pull/47983)
```
Comment (afc163) [inline CHANGELOG.en-US.md:28]:
```suggestion
  - 🤖 Fix the ts util `GetRef` can not get correct ref type for some component. [#47983](https://github.com/ant-design/ant-design/pull/47983)
```

Comment (zombieJ) [reply CHANGELOG.en-US.md:28]: MISC 标记需要加上，否则 CI 过不了

Comment (chenshuai2144) [reply CHANGELOG.en-US.md]: 每一个字符都有他的意义

**Review by zombieJ — APPROVED** [review]

**Pull request merged**